### PR TITLE
[FR] Fallback Option for Location/ReverseLookup

### PR DESCRIPTION
**Pull request by danner26**
Added support and CLI switch for fallback. If Apple lookup fails, and if the command is specified, it will try to redo the lookup through Google. Likewise, if the Apple reverse lookup fails, and if the command is specified, it will try to redo the reverse lookup through Google.

### CODE REVIEW

File: pkgroot/Library/Application Support/pinpoint/bin/pinpoint [hunk 1]
```diff
@@ -776,6 +820,11 @@ def main():
     else:
         address_lookup_service = pref('AddressLookupService').lower()
 
+    if args.fallback:
+        toFallback = True
+    else:
+        toFallback = False
```
Comment (clburlison) [inline pkgroot/Library/Application Support/pinpoint/bin/pinpoint]: You should pull from the config here. `pref('Fallback')`

**Comment by danner26** [comment]
Whoops, missed that one! Thanks for catching, I updated it.

**Comment by danner26** [comment]
Did you have a chance to review this? if so, it would be great to have it in the master branch.

**Comment by clburlison** [comment]
@danner26 I just got time to review this. I'm happy with the feature enhancement addition but not the implementation. Using exceptions for core logic handling is pretty dirty. Since I was delayed on the review I've gone ahead and refactored this. Please see #33 and see if you're happy with the changes I've made. If so I'll go ahead with the merged and cut a new patch.

**Pull request closed without merging**